.github/actions/npm-rc-release/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Npm RC release'
+description: 'Publish code changes to npmjs.org'
+inputs:
+  git_email:
+    description: 'Email of git commit containing version bump'
+    required: true
+  git_username:
+    description: 'Commit author name'
+    required: true
+  npm_token:
+    description: 'Npm token for publishing. Login to npmjs.org to generate this.'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Configure git CLI'
+      shell: bash
+      run: |
+        git config --global user.email "${{ inputs.git_email }}"
+        git config --global user.name "${{ inputs.git_username }}"
+    - name: 'Bump version'
+      shell: bash
+      run: npm version prerelease --preid=rc --no-git-tag-version
```

Comment:
Can we make the "rc" here configurable through the workflow? 

So that someone creating an RC could pass in something like "rc.2024-07" for example

Reply:
yes!